integration-tests/testutils/build/build.go

```diff
@@ -51,10 +51,12 @@ var (
 	// integration-tests/test-wrapper script (Test-Command's entry point of
 	// adt-run) takes care of including testsBinDir at the beginning of $PATH, so
 	// that these binaries (if they exist) take precedence over the system ones
-	buildSnappyCliCmd = "go build -tags=excludeintegration -o " +
+	buildSnappyCliCmd = "go build -o " +
 		filepath.Join(testsBinDir, "snappy") + " ." + string(os.PathSeparator) + filepath.Join("cmd", "snappy")
-	buildSnapdCmd = "go build -tags=excludeintegration -o " +
```

Comment:
@fgimenez no, the actual compiled main package is what go build is pointed at, is not all things in main packages around